brainframe/api/stubs/capsules.py

```diff
@@ -28,6 +29,38 @@ def get_capsules(self, timeout=DEFAULT_TIMEOUT) -> List[Capsule]:
         capsules, _ = self._get_json(req, timeout)
         return [Capsule.from_dict(d) for d in capsules]
 
+    def set_capsule(self, storage_id: int,
```

Comment:
Is `set` the best term to use here? Would `load` be better?

Reply:
I had trouble with this too. "Set" is the term we use for all these `PUT` methods, so I was trying to be consistent, but I agree that it's not a good name. I'll change it to `load_capsule`.